translations/de/info/task_description.html

```diff
@@ -1,66 +1,78 @@
+<html>
+    <meta charset="utf-8"
+</html>
 <div style="text-align: center">
         <img src="{{MEDIA}}title.png" style="max-height: 350px; max-width: 100%">
 </div>
 
 <p>
-    Roman numerals come from the ancient Roman numbering system.
-    They are based on specific letters of the alphabet which are combined to signify the sum
-    (or, in some cases, the difference) of their values. The first ten Roman
-    numerals are:
+    Die römische Zahlen stammen aus dem alten römischen Zahlensystem.
+    Sie basieren auf bestimmten Buchstaben des Alphabets.
+    Addiert man die Werte der römischen Ziffern, ergibt die Summe (Oder,
+    in einigen Fällen die Differenz) die Zahl.
+</p>
+<p>Die ersten zehn römischen Zahlen lauten:
+    I, II, III, IV, V, VI, VII, VIII, IX, und X.
 </p>
-<p>I, II, III, IV, V, VI, VII, VIII, IX, and X.</p>
 <p>
-    The Roman numeral system is decimal based but not directly positional and does not
-    include a zero. Roman numerals are based on combinations of these seven symbols:
+    Das römische Zahlensystem basiert auf den Zehnerpotenzen als Basis und den
+    fünffachen Halbbasiswerten. Abgesehen von der Subtraktionsregel ist der Wert
```

Comment:
I really meant _fünffachen Halbbasiswerten_, which is correct a 5 is the half of 10... :sunglasses: 
But I prefer your version.
:+1: